src/magicbox/MagicBox.ts

```diff
@@ -204,7 +204,7 @@ export class MagicBoxInstance {
     const suggestions = await this.suggestionsManager.receiveSuggestions(this.getSuggestions != null ? this.getSuggestions() : []);
     this.addSelectEventHandlers(suggestions);
     this.inputManager.setWordCompletion(this.firstSuggestionText);
-    this.onSuggestions(suggestions);
+    this.onSuggestions && this.onSuggestions(suggestions);
```

Comment:
This was added to prevent an error that I saw during unit tests. By reflex, I thought `onSuggestions` was meant to be optional and I made it optional. However, looking back, the error must've been caused by unit tests not setting the value of `onSuggestions`. I'll revert those two lines and fix unit tests instead.